src/bentoml/_internal/runner/runner.py

```diff
@@ -240,6 +241,12 @@ def init_client(self):
     def destroy(self):
         object.__setattr__(self, "_runner_handle", DummyRunnerHandle())
 
+    async def runner_handle_is_ready(
```

Comment:
We forgot to add this so `Provide` actually works.

```suggestion
    @inject
    async def runner_handle_is_ready(
```